docker-compose.yml

```diff
@@ -91,6 +91,8 @@ services:
       - bring-emulator
     volumes:
       - ./configuration/integration-layer/config.json:/usr/src/app/config.json
+      - ./configuration/integration-layer/logging.json:/usr/src/app/logging.json
```

Comment:
You can mount the whole config directory

Reply:
Done